app/javascript/components/maps/components/basemaps/basemaps-component.jsx

```diff
@@ -50,11 +54,14 @@ class Basemaps extends React.PureComponent {
   renderDropdownBasemap(item) {
     const { selectBasemap, activeBasemap, landsatYears, basemaps } = this.props;
     const year = activeBasemap.year || landsatYears[0].value;
+    const basemap = basemaps[item.value]
+      ? basemaps[item.value]
+      : basemaps.landsat;
```

Comment:
This should be agnostic of what basemap there is. Ideally we don't want to reference landsat at all, we should read from the config so we can repeat for another basemap in the future.